### DISCUSSION TITLE
Summer/Fall 2021 Maintenance!!

### DISCUSSION THREAD

**Discussion by zsunberg** [JuliaPOMDP/POMDPs.jl#357] [General]
Alright, I am finally going to put in some effort at long-overdue maintenance on this package! If you are interested in helping with a particular task, please let me know which task and/or comment in the individual issues.

There are four major steps that I plan to take in this order:
1. ~~Make sure that the appropriate github actions are installed (#356)~~
2. Consolidate the tools packages (#290)
3. Improve Documentation([documentation label](https://github.com/JuliaPOMDP/POMDPs.jl/issues?q=is%3Aopen+is%3Aissue+label%3Adocumentation))
4. Release POMDPs.jl version 1.0

@bkraske @whifflefish @MaximeBouton @Shushman @rejuvyesh @mossr @lassepe thought you might have suggestions for other issues that need to be fixed or be able to help with something. Please tag anyone else who might be interested.

**Comment by zsunberg** [comment]
@qhho @himanshugupta1009

**Comment by zsunberg** [comment]
Fixing this is also super important: https://github.com/JuliaPOMDP/quickpomdps I want to make that work well (this will require some python ecosystem knowledge, so help is especially needed here)

**Reply by rejuvyesh** [reply]
I can work on this. Let me see what's the current best practice.

**Reply by rejuvyesh** [reply]
One question is what should we pre-install? All the related packages under  `JuliaPOMDP` org. Add a function that does the package management on the julia side from python?

**Reply by zsunberg** [reply]
@rejuvyesh thanks! I don't think we should install any solvers by default. Rather, we should give instructions (I don't think it will be too hard to do `from julia import Pkg; Pkg.add("QMDP")` or something like that.

The most important thing to add is testing. Currently QuickPOMDP doesn't work because of some quirks that I added surrounding `reward` and `observation`. I don't think this should be difficult to fix with `QuickPOMDPs.preprocess(::PyObject)`.
